docs/SlavaUkraini/examples/accessibility/index.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: tutorial_v2
+title: A guide to basic Leaflet accessibility
+---
+
+## Accessible maps
+
+Web accessibility is the inclusive practice of ensuring no barriers exist
+that would prevent interactions or information access.
+
+This guide to Leaflet accessibility can help you create maps that are usable
+to persons of a wide range of abilities.
+
+### Preserve useful defaults
+
+Leaflet comes with a set of useful defaults.
+
+The map container and markers are keyboard operable by default,
+this enables users who are unable to use a pointing device.
+Consider the effects on your users before changing defaults such as these.
+
+### Markers must be labelled
+
+When using markers,
+it is vital to ensure each has a unique and descriptive
+[`alt`](/reference.html#marker-alt)
+or
+[`title`](/reference.html#marker-alt):
+
+<pre><code>var marker = L.marker([50.4501, 30.5234],
```

Comment:
This is a markdown document, so code blocks should be delimited by three backticks (```) instead of HTML.

Reply:
For this particular code block I'm using a link in a comment (on the next line), so I think this is required. This is also done on other example.md pages.